### PR TITLE
Virtualbox repository enabled and RPM-GPG-KEY-Virtualbox 

### PR DESCRIPTION
**Pull request by stephdl**
- Enable the virtualbox repository with software-repos-save
- The GPG key of virtualbox is provided inside our rpm
- make a nethgui widget for the web link

https://github.com/NethServer/dev/issues/5826

### CODE REVIEW

File: VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```
Comment (gsanchietti) [inline VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo:2]: License header is missing

File: VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+echo '[NOTICE] applying new YUM repository configuration'
+exec /sbin/e-smith/signal-event software-repos-save
```
Comment (gsanchietti) [inline VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo:21]: I do not like to call the software-repo-save from within an update event, but it seems we already do it eve in base: https://github.com/NethServer/nethserver-base/blob/master/root/etc/e-smith/events/actions/nethserver-base-fixrepos

File: nethserver-virtualbox.spec [hunk 3]
```diff
@@ -107,6 +107,9 @@ done
 %pre phpvirtualbox
 
 %changelog
+* Mon Apr 15 2019 Stephane de Labrusse <stephdl@de-labrusse.fr> - 1.0.2-1
```
Comment (gsanchietti) [inline nethserver-virtualbox.spec]: This changelog is not relevant to this PR (nor the version increment).
Am I right?

Comment (stephdl) [reply nethserver-virtualbox.spec]: yes :), something new in QA I need to ask for test

File: VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo (hunk 1 above)
Comment (stephdl) [reply VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo:2]: ok

File: VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo (hunk 2 above)
Comment (stephdl) [reply VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo:21]: ok

File: nethserver-virtualbox.spec (hunk 3 above)
Comment (stephdl) [reply nethserver-virtualbox.spec]: I removed the commit

File: phpvirtualbox/usr/share/nethesis/NethServer/Module/Dashboard/Applications/PhpVirtualBox.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+ * NethServer is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/**
+ * SOGo web interface
```
Comment (gsanchietti) [inline phpvirtualbox/usr/share/nethesis/NethServer/Module/Dashboard/Applications/PhpVirtualBox.php]: This is not SOGo :D

File: VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo [hunk 5]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2019 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
```
Comment (gsanchietti) [inline VirtualBox/etc/e-smith/events/actions/virtualbox_enablerepo]: Change to your contacts or just remove the line.

**Pull request merged**